layouts/partials/hooks/body-end.html

```diff
@@ -2,3 +2,5 @@
   {{ partial "script.html" (dict "src" "js/tracing.js") -}}
 {{ end -}}
 {{ partial "script.html" (dict "src" "js/navScroll.js") -}}
+{{ $js := resources.Get "js/scrollToTop.js" | minify | fingerprint }}
+<script src="{{ $js.Permalink }}"></script>
```

Comment:
We don't want to unconditionally minify and fingerprint (those should be done in production only). I'd rather that this be part of the main JS script. But I'll have to get back to you on that.

Reply:
Actually, just use the `{{ partial "script.html" ... }}` partial as is done for the other .js files in the lines above this one.